frontend/src/metabase/collections/components/ModelUploadModal.tsx

```diff
@@ -0,0 +1,116 @@
+import { useState, useMemo, useEffect } from "react";
+import { t } from "ttag";
+import {
+  Modal,
+  Button,
+  Flex,
+  Stack,
+  Select,
+  Radio,
+  Text,
+  Icon,
+} from "metabase/ui";
+import { useSearchListQuery } from "metabase/common/hooks";
+import type { CollectionId, TableId, CardId } from "metabase-types/api";
+
+type UploadMode = "append" | "create";
```

Comment:
!nit (<- meaning don't feel like you have to make this change)

```suggestion
enum UploadMode {
  append = "append",
  create = "create",
}
```

Then you can reference the enum below to avoid having magic strings below.

Reply:
well TIL https://www.typescriptlang.org/docs/handbook/enums.html#enums-at-runtime